docs/json_schemas/vault_transit/v0/provider.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "ProviderSchema",
+  "description": "The schema for the provider side of this interface.",
+  "type": "object",
+  "properties": {
+    "unit": {
+      "$ref": "#/definitions/BaseModel"
+    },
+    "app": {
+      "$ref": "#/definitions/VaultTransitProviderSchema"
+    }
+  },
+  "required": [
+    "app"
+  ],
+  "definitions": {
+    "BaseModel": {
+      "title": "BaseModel",
+      "type": "object",
+      "properties": {}
+    },
+    "VaultTransitProviderSchema": {
+      "title": "VaultTransitProviderSchema",
+      "type": "object",
+      "properties": {
+        "address": {
+          "title": "Address",
+          "description": "The address of the Vault server to connect to.",
+          "type": "string"
+        },
+        "token_secret_id": {
```

Comment:
Here we call it `token_secret_id` but in the example in the readme we call it `credentials_secret_id`

Reply:
This file was leftover from before the re-spec. It was a generated file so I didn't even notice it. Will remove.

Thanks for catching this.